src/Header/Search/__snapshots__/Search.test.jsx.snap

```diff
@@ -33,5 +33,11 @@ exports[`Search Rendering Matches snapshot 1`] = `
       search
     </span>
   </button>
+  <a
+    className="link"
+    href="#undefined"
```

Comment:
I think this test would be better to pass in a `skipLinkId` prop so that the snapshot makes more sense